shell/app-shell/app-shell.js

```diff
@@ -215,8 +217,11 @@ const template = ArcsUtils.html`
         <option value='{{value}}' selected='{{selected}}'>{{user}}</option>
       </template>
       <div buttons>
+        <toggle-button title='{{shareStateTitle}}' state='{{shareState}}' on-state='_onShareState' icons='share person people'></toggle-button>
+        <!--
```

Comment:
Should the commented out section be removed?

Reply:
I sometimes want to leave some code around for an iteration or two for reference. I'll remove it before the new sharing changes are finished, if that's ok.

Reply:
Sounds good to me.